src/main/java/frc/robot/commands/autos/OneFiveLeave.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.autos;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.commands.IntakeSequence;
+import frc.robot.commands.autos.components.DriveDistance;
+import frc.robot.commands.autos.components.ScoreThisNote;
+import frc.robot.subsystems.conveyor.Conveyor;
+import frc.robot.subsystems.drive.DriveConstants;
+import frc.robot.subsystems.drive.DriveSubsystem;
+import frc.robot.subsystems.elevator.Elevator;
+import frc.robot.subsystems.intake.Intake;
+import frc.robot.subsystems.shooter.Shooter;
+
+/** score the preloaded note into the speaker, then get another note, return to home state */
+public class OneFiveLeave extends SequentialCommandGroup {
+  public OneFiveLeave(Intake intake, Elevator elevator, Shooter shooter, Conveyor conveyor, DriveSubsystem drive) {
+    final double X_DISTANCE_TO_NOTE = Constants.PLACEHOLDER_DOUBLE;
```

Comment:
Can you guesstimate this distance please?

Reply:
Also, add units